src/main/play-doc/general/ReleaseNotes.md

```diff
@@ -1,6 +1,27 @@
 # ConductR release notes
 
 
-## 1.2.0
+## 2.0.0 Alpha
+
+### Completed
+
+* Separate replication/scaling concerns into ConductR Core and bundle executions concerns into ConductR Agent.
+* Package and deploy ConductR as a framework within Mesos or DC/OS context.
+* Implement support for running and scaling of ConductR bundles when deployed within Mesos or DC/OS context.
```

Comment:
loading, running and scaling of ConductR JVM bundles (Docker bundles are currently untested)